app/components/course-overview.js

```diff
@@ -29,6 +29,14 @@ export default Ember.Component.extend({
       promise: deferred.promise
     });
   }.property(),
+
+  externalIdValidations: {
+    'buffer': {
```

Comment:
I'm not a huge fan of buffer here.  Maybe we can use a oneWay computed value on the mixin and then instead of buffer we can use userInput here?  Don't need to make the change in this version, but think about it for when we create a lot more validators.